modules/confiantRtdProvider.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * This module provides comprehensive detection of security, quality, and privacy threats by Confiant Inc,
+ * the industry leader in real-time detecting and blocking of bad ads
+ *
+ * The {@link module:modules/realTimeData} module is required
+ * The module will inject a Confiant Inc. script into the page to monitor ad impressions
+ * @module modules/cleanioRtdProvider
+ * @requires module:modules/realTimeData
+ */
+
+import { submodule } from '../src/hook.js';
+import { logError, insertElement, generateUUID } from '../src/utils.js';
+import * as events from '../src/events.js';
+import CONSTANTS from '../src/constants.json';
+
+/**
+ * Injects the Confiant Inc. configuration script into the page, based on proprtyId provided
+ * @param {string} propertyId
+ */
+function injectConfigScript(propertyId) {
+  const scriptElement = document.createElement('script');
+  scriptElement.type = 'text/javascript';
+  scriptElement.src = `https://cdn.confiant-integrations.net/${propertyId}/gpt_and_prebid/config.js`;
+
+  insertElement(scriptElement);
```

Comment:
you must use our methods, please import the ajax utility

Reply:
@patmmccann this is pretty much the same approach that Clean.io is taking to inject their script. Do you mean we should use ajax to fetch .js and the `eval()` it?

Reply:
@patmmccann just like Clean.io our client asked to implement this to cover other demand as well native, GPT as well if they choose to do so.
So we are just replicating the approach that was approved for the Clean.io prebid module, see the link above.

Reply:
They are not similar and the change request stands.

Reply:
I looked at this again; the reason I flagged them as not similar is bc you linked the wrong part of the clean adapter, preloading is not against the rules (although we are considering it). The correct part of the clean adapter that inserts has been added to this list https://github.com/prebid/Prebid.js/issues/8421

Reply:
@denys-berzoy-confiant it seems that the new way of inserting scripts is this: https://github.com/prebid/Prebid.js/pull/8496/files#diff-daa702a1f45a1ffc6ff61f98bfd8b733621e125a5525b3c2927e53294f2a0fd5R14
please update the PR

Reply:
this is resolved now ty